uuid-types/tests/BenchUUID.hs

```diff
@@ -6,7 +6,7 @@
 import           Control.DeepSeq
 import qualified Data.ByteString.Lazy.Internal as BL
 #endif
-import           Criterion.Main
+import           Gauge.Main
```

Comment:
I'm quite sure @hvr won't depend on `gauge`.

Reply:
I'm quite serious.

Reply:
In that case, he might want to extract the benchmarks into a separate package or handle the circular dependency of criterion on uuid in some other way.

I personally don't care about uuid package, if I ever need it I can always make a fork of it.

Reply:
@lehins Most likely you will be fine with using `uuid-types`'s `UUID`.

Reply:
> I'm quite sure @hvr won't depend on gauge.

This is a very strange "stubbornness" if you ask me. I am the second person to implement this switch to gauge completely independently (see #51) and there seems to be not even a single comment from the maintainer (@hvr) about this.

I personally prefer criterion as well, but it is problematic for packages that criterion depends on. There is discussion in vector to switch to gauge because of this as well. haskell/vector#321

In any case, as I said, I don't personally care about uuid or uuid-types so I won't get upset if this PR gets bit rotted.